ios/app/brave_core_main.h

```diff
@@ -84,6 +85,26 @@ OBJC_EXPORT
 
 @property(readonly) BraveStats* braveStats;
 
+/// The main shields file install path (KVO compiliant)
```

Comment:
Sure yeah I can do that, it means having to pass the component updater in to avoid reaching backwards via GetApplicationContext I believe but should be fine

Reply:
Done in https://github.com/brave/brave-core/pull/14248/commits/db6180b54b5733e1bfed7efa8f5e4121c472dfbd